hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/TokenHandlerHelper.java

```diff
@@ -152,7 +151,7 @@ public static Account getIfUsable(
         validateTrue(acct != null, errorIfNotUsable);
         final var isContract = acct.smartContract();
 
-        validateFalse(acct.deleted(), isContract ? CONTRACT_DELETED : errorOnAccountDeleted);
```

Comment:
Wouldn't smart contract service reach this code ?

Reply:
To match what mono's return.